docs/pages/use-teleport/tsh.mdx

```diff
@@ -764,5 +764,55 @@ $ tsh status
   Extensions:         permit-X11-forwarding
 ```
 
+## Custom aliases and defaults
+
+You can define your own custom commands and set command-specific flag defaults by using the `aliases` section of `tsh` config.
+
+There are two configuration files:
+- global: by default `/etc/tsh.yaml`, unless overridden by `$TELEPORT_GLOBAL_TSH_CONFIG` env var.
+- user-specific: `$TELEPORT_HOME/config/config.yaml`, which by defaults resolves to `~/.tsh/config/config.yaml`.
+
+The user-specific config is merged with global config. In case of conflicts (i.e. same alias defined in both files), the user-specific config has higher priority.
+
+In either of those files you can add define an alias such as:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
+```
+
```

Comment:
As with the config reference, I think we should include more general information about how `tsh` handles the alias config. Does each key become a separate subcommand? Are there any restrictions on keys?